docs/UserGuide.md

```diff
@@ -207,40 +189,40 @@ Format: `delete -TYPE`
 
 Deletes the specified Applicant from HireLah.
 
-Format: `del -a INDEX`
+Format: `del -a CANDIDATE_INDEX`
 
-* Deletes the Applicant at the specified `INDEX`.
+* Deletes the Applicant at the specified `CANDIDATE_INDEX`.
 * The index refers to the index number shown in the displayed Applicant list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
 * `list` followed by `delete -a 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete -a 1` deletes the 1st person in the results of the `find` command.
 
+### Deleting an Interview: `del -i`
+
+Deletes an existing interview in HireLah.
+
+Format: `del -i CANDIDATE_INDEX ROLE`
```

Comment:
I have changed the format of the interview command, I will fix it later when it's finalised!